{"id": "ROOT", "speaker": "penn5", "conversation_id": "ROOT", "reply_to": "ROOT", "timestamp": "2018-11-05T11:33:47Z", "text": "Push: NoneType has no attribute Write I am getting the error: \r ```\r b'device'\r Traceback (most recent call last):\r   File \"autoroot.py\", line 123, in <module>\r     install_firmware(dloadfirmware, device)\r   File \"autoroot.py\", line 120, in install_firmware\r     print(device.Push(os.path.join(d, 'Software', 'dload'), storage+'/dload', timeout_ms=100000))\r   File \"/home/penn/python-adb/adb/adb_commands.py\", line 269, in Push\r     progress_callback=progress_callback)\r   File \"/home/penn/python-adb/adb/adb_commands.py\", line 280, in Push\r     mtime=int(mtime), progress_callback=progress_callback, **kwargs)\r   File \"/home/penn/python-adb/adb/filesync_protocol.py\", line 149, in Push\r     cnxn.Send(b'DATA', data)\r   File \"/home/penn/python-adb/adb/filesync_protocol.py\", line 207, in Send\r     self._Flush()\r   File \"/home/penn/python-adb/adb/filesync_protocol.py\", line 254, in _Flush\r     self.adb.Write(self.send_buffer[:self.send_idx])\r AttributeError: 'NoneType' object has no attribute 'Write'\r ```\r \r I've confirmed that the path I'm pushing to exists. I'm pushing a directory to another directory. I'm using Ubuntu 18.04. I am able to use the Shell command fine.\r \r This looks similar to #125 except in Push not Pull.\r \r The adbd is running as root on the phone, as is python3 on my PC.", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM0", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "ROOT", "timestamp": "2018-11-13T08:51:22Z", "text": "Hello, what is your version Android me it's 8.1 ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM1", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM0", "timestamp": "2018-11-13T16:20:42Z", "text": " Look here for me solve my problem temporary https://github.com/google/python-adb/issues/132 ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM2", "speaker": "caffeinatedMike", "conversation_id": "ROOT", "reply_to": "COM1", "timestamp": "2018-11-13T16:55:13Z", "text": "@VBlackOut Read @fahhem's response. That is not the way to solve the issue. \n\nAlso, something odd is happening, your replies are either getting auto-deleted on threads (likely because of the repetitive comments) or you're deleting them yourself. For my Issue you posted the same comment and NO, that does not fix the problem and NO that is not the answer to the bug. ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM3", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM2", "timestamp": "2018-11-13T17:08:43Z", "text": "It's just re-identify the id if change ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM4", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM3", "timestamp": "2018-11-13T17:09:15Z", "text": "Not best practice but it's works", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM5", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM4", "timestamp": "2018-11-13T17:11:22Z", "text": "Sorry I use GitHub on phone (fasthub) it's not ideal for comment and explore the interface GUI", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM6", "speaker": "caffeinatedMike", "conversation_id": "ROOT", "reply_to": "COM5", "timestamp": "2018-11-13T17:16:00Z", "text": "I use the same app and have zero issues with comments, etc. Also, your explanation still makes zero sense \"It's just re-identify the id if change\" isn't even a proper sentence. And even after being told by the main contributor of the repo that that is not a fix you continue to spam peoples' posts with this. Just PLEASE STOP. Listen to his comment and stop suggesting this crudely thought-up bandaid of a \"fix\". ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM7", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM6", "timestamp": "2018-11-13T17:32:37Z", "text": "It's just forward fix but it's not finally real fix ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM8", "speaker": "vBlackOut", "conversation_id": "ROOT", "reply_to": "COM7", "timestamp": "2018-11-13T17:33:01Z", "text": "I just for understand the code ", "meta": {"posReactions": "0", "negReactions": "0"}}
{"id": "COM9", "speaker": "fahhem", "conversation_id": "ROOT", "reply_to": "COM8", "timestamp": "2018-11-13T17:44:37Z", "text": "Whoa let's all calm down please. @vBlackOut is just trying to help (and clearly english isn't his first language, so give him a break), it seems you're a little too caffeinated here :)\r\n\r\nLet's start over in a new issue to avoid this exchange.", "meta": {"posReactions": "0", "negReactions": "0"}}
